websites/T/TruckersMP/dist/metadata.json

```diff
@@ -1,30 +1,30 @@
 {
   "author": {
-    "name": "Bas950",
-    "id": "241278257335500811"
+    "name": "Encrypted",
+    "id": "564434085708038144"
   },
   "service": "TruckersMP",
   "description": {
-    "en": "TruckersMP a Multiplayer Truckers experience for SCS Software's Euro Truck Simulator 2 and American Truck Simulator.",
-    "nl": "TruckersMP een Multiplayer Truckers ervaring voor SCS Software's Euro Truck Simulator 2 en American Truck Simulator."
+    "en": "TruckersMP is a multiplayer mod available for Euro Truck Simulator 2 and American Truck Simulator"
```

Comment:
this is their official description so why  change it?

Reply:
Changed it to the one from here: 
![image](https://user-images.githubusercontent.com/64903135/111686287-ca579300-8820-11eb-9830-33e8c1c5c6bf.png)

Reply:
The one it has/had is the one of their website and presence support websites so no need to change.